content/pt-br/docs/concepts/scheduling-eviction/_index.md

```diff
@@ -1,8 +1,32 @@
 ---
-title: "Escalonamento"
-weight: 90
+title: "Escalonamento, preempção e remoção"
+weight: 95
+content_type: concept
 description: >
-   No Kubernetes, agendamento refere-se a garantia de que os pods correspondam aos nós para que o kubelet possa executá-los.
-   Remoção é o processo de falha proativa de um ou mais pods em nós com falta de recursos.
+  No Kubernetes, escalonamento refere-se à certeza de que os Pods correspondam aos nós para que o kubelet possa executá-los. Preempção é o processo de finalizar Pods com menor prioridade, para que os Pods com maior prioridade possam ser escalonados nos nós. Remoção é o processo de finalização proativa de um ou mais Pods em nós com poucos recursos.
```

Comment:
Minor: the "poucos recursos" is not present in the original doc (sentence reads "Eviction is the process of terminating one or more Pods on nodes").

Reply:
Hi! Thanks for the feedback. Necessary changes done! Merry Christmas!

Reply:
Merry Christmas! 🎄